src/ydata_synthetic/synthesizers/regular/cgan/model.py

```diff
@@ -172,8 +172,8 @@ def __init__(self, batch_size, num_classes):
     def build_model(self, input_shape, dim, data_dim):
         noise = Input(shape=input_shape, batch_size=self.batch_size)
         label = Input(shape=(self.num_classes,), batch_size=self.batch_size, dtype='int32')
-        label_embedding = Embedding(2, 1)(label)
-        input = Flatten()(multiply([noise, label_embedding]))
+        label_dense = Dense(1)(label)
```

Comment:
Changed Embedding layer to Dense layer with same dimension, numerically outputs are expected to be the same. We should be able to process both numeric and categoric conditions this way. [Computationally should be more expensive.](https://stackoverflow.com/questions/47868265/what-is-the-difference-between-an-embedding-layer-and-a-dense-layer)